client/packages/invoices/src/OutboundShipment/DetailView/OutboundLineEdit/allocation/allocateQuantities.test.ts

```diff
@@ -141,10 +158,41 @@ describe('Allocated quantities - coping with over-allocation', () => {
     expect(remainingQuantity === 0).toBe(true);
   });
 
+  it('reduces correct quantities based on pack size', () => {
```

Comment:
found a bug in existing allocation logic 😱 weren't considering rounding to pack size in the "reduceBatches" function, below test was failing, it actually under-allocated!